core/src/main/scala/org/apache/spark/api/java/JavaRDDLike.scala

```diff
@@ -62,6 +62,9 @@ trait JavaRDDLike[T, This <: JavaRDDLike[T, This]] extends Serializable {
   /** Set of partitions in this RDD. */
   def partitions: JList[Partition] = rdd.partitions.toSeq.asJava
 
+  /** Return the number of partitions in this RDD. */
```

Comment:
Not at all. Should I use `@since` or `@Since`, and also add it to RDD.scala? (I could not find any occurrence of this tag in spark-core).